src/xrpld/app/tx/detail/Transactor.cpp

```diff
@@ -831,6 +831,24 @@ removeExpiredCredentials(
     }
 }
 
+static void
+modifyWasmDataFields(
+    ApplyView& view,
+    std::vector<std::pair<uint256, Blob>> const& wasmObjects,
+    beast::Journal viewJ)
+{
+    for (auto const& value : wasmObjects)
```

Comment:
auto const &[index, data]: wasmObjects

Reply:
Fixed in ee76920b839f7e29aa749adfd22efea8d237e697